securedrop_client/export.py

```diff
@@ -117,7 +117,7 @@ def connect_signals(
                 self.run_printer_preflight, type=Qt.QueuedConnection
             )
 
-    def _export_archive(cls, archive_path: str) -> ExportStatus:
+    def _export_archive(cls, archive_path: str) -> Optional[ExportStatus]:
```

Comment:
@rocodes I suppose you're going for the minimal change set to fix the bug, which seems very fair.

I'd prefer adding a success value to the `ExportStatus` and asserting on that where relevant, but I think that's a matter of personal preference. I feel like `ExportStatus` is broad enough as a name to accommodate a success status, and it feel to me that defining an `enum` that doesn't cover all possible outcomes is surprising; but again I don't think that's a question that needs to be resolved while fixing this bug.

Reply:
Replied in https://github.com/freedomofpress/securedrop-client/pull/1596#discussion_r1028556760 :+1:

Reply:
I hear ya. My reasoning is twofold: 1, there are several spots in the [existing code](https://github.com/freedomofpress/securedrop-client/blob/main/securedrop_client/export.py#L229)) that presuppose the current API.  2, since sd-devices currently returns the empty string for 3 different successes (export, print, print preflight), and will continue to do so until we start using the new export code, it didn't make sense to me to try to make up a new `ExportStatus` enums for a bugfix, especially since it will be deprecated almost immediately by the more precise status values we have in place for the new export code.